playbook/CHANGELOG.md

```diff
@@ -18,7 +19,8 @@ You can now wrap our Date kit with any of our typography kits (Title, Body, Capt
 - Changelog Improvements [\#2591](https://github.com/powerhome/playbook/pull/2591) ([jasoncypret](https://github.com/jasoncypret))
 - Separation of Markdown & PBDocs Dependencies from Playbook Library [\#2527](https://github.com/powerhome/playbook/pull/2527) ([jasoncypret](https://github.com/jasoncypret))
 
-# Responsive Components in Rapid Time: Introducing Global Responsive Spacing Props for React and Rails Components. 
+# Responsive Components in Rapid Time: Introducing Global Responsive Spacing Props for React and Rails Components.
+##### May 26, 2023
```

Comment:
I have a question: Should the date on the changelog be the date the release is actually cut and pushed up to npm and rubygems, or should it be the date the release is merged/announced to Nitro? I would assume it would be the date the release was actually cut since that was the way we did it before moving to this new way of doing to changelog? May 26th is the date 12.24.0 was cut, but for 12.25.0 above this, the date the release was cut was June 9th. June 13th was the date it was merged/announced to Nitro @jasperfurniss @jasoncypret